database/index.js

```diff
@@ -1,22 +1,20 @@
 const mongoose = require('mongoose');
 
 const { Schema } = mongoose;
+mongoose.Promise = global.Promise;
```

Comment:
Is this line necessary? It looks like Mongoose 5.0 uses native promises by default. https://mongoosejs.com/docs/4.x/docs/api.html#promise_Promise